compiler/luci/import/src/Nodes/CircleConst.cpp

```diff
@@ -150,6 +102,33 @@ void copy_data<loco::DataType::STRING>(const VectorWrapper<uint8_t> &raw_data,
   }
 }
 
+// NOTE copy_data for S4, U4
+template <loco::DataType DT>
+void copy_data_4(const VectorWrapper<uint8_t> &raw_data, uint32_t num_elements,
+                 CircleConst *const_node)
+{
+  using T = typename loco::DataTypeImpl<DT>::Type;
+
+  // TODO support sparse?
+  assert(const_node->sparsityparam() == nullptr);
+  if (const_node->sparsityparam())
+    return;
+
+  uint32_t raw_size = (num_elements + 1) / 2;
+  assert(raw_data.size() == raw_size);
+
+  const uint8_t *data = raw_data.data();
+  const_node->size<DT>(num_elements);
+  for (uint32_t i = 0; i < raw_size; ++i)
+  {
+    uint32_t idx = i * 2;
+    // 1bit for sign, 3bit for value
```

Comment:
Can this comment apply to U4 as well? I guess 4 bit for value when it comes to the U4.